packages/lab/src/tabs/Tab.css

```diff
@@ -27,6 +27,13 @@
   user-select: none;
 }
 
+/* Overrides characteristic used in uitkFocusVisible */
+.uitkTab.uitkFocusVisible:after,
```

Comment:
Bit confused why .focused is being used here?

Reply:
hmm, i was ignorant and thought it would apply here since thats how it is in the focused.css, but digging into code quickly im not sure why its there either. removed it from here at least